java/client/src/org/openqa/selenium/remote/SimplePropertyDescriptor.java

```diff
@@ -24,14 +25,16 @@
   private String name;
   private Method readMethod;
   private Method writeMethod;
+  private Field field;
 
   public SimplePropertyDescriptor() {
   }
 
-  public SimplePropertyDescriptor(String name, Method readMethod, Method writeMethod) {
+  public SimplePropertyDescriptor(String name, Method readMethod, Method writeMethod, Field field) {
```

Comment:
Since you pass in null as the last parameter a few times (at least in this class), why not create another constructor?